Controller/LoginController.php

```diff
@@ -64,8 +71,45 @@ public function loginAction(Request $request)
         return $this->templating->renderResponse(
             'SumoCodersFrameworkMultiUserBundle:Login:login.html.twig',
             [
-                'error' => $exception ? $exception->getMessage() : null,
+                'error' => $this->getTranslatedErrorMessageFromAuthenticationException($exception),
             ]
         );
     }
+
+    /**
+     * @param AuthenticationException|null $exception
+     *
+     * @return null|string
+     */
+    private function getTranslatedErrorMessageFromAuthenticationException(AuthenticationException $exception = null)
+    {
+        if ($exception === null) {
+            return null;
+        }
+
+        switch (true) {
```

Comment:
heb dit nog nooit gezien, ik vind het maar vreemd dat je een true in een switch moet steken, meerdere ifs zou logischer zijn, maar laat maar zo zeker